include/swift/SIL/SILInstruction.h

```diff
@@ -3474,11 +3474,12 @@ class HopToExecutorInst
 {
   friend SILBuilder;
 
-  HopToExecutorInst(SILDebugLocation DebugLoc, SILValue Actor, bool HasOwnership)
-      : UnaryInstructionBase(DebugLoc, Actor) { }
+  HopToExecutorInst(SILDebugLocation debugLoc, SILValue executor,
+                    bool hasOwnership)
+      : UnaryInstructionBase(debugLoc, executor) { }
 
 public:
-  SILValue getActor() const { return getOperand(); }
+  SILValue getTargetExecutor() const { return getOperand(); }
```

Comment:
Can you add a short note in SIL.rst for ``hop_to_executor``, describing the two operand variants and that it gets lowered from Actor -> Executor?

Reply:
Done